common/package.yaml

```diff
@@ -0,0 +1,24 @@
+name:                common
+version:             0.1.0.0
+#synopsis:
+#description:
+homepage:            https://github.com/DebugSteven/coleg-scraper/common
+license:             BSD3
+author:              J Haigh
+maintainer:          jslhaigh@gmail.com
+copyright:           2017 J Haigh
+category:            Web
+extra-source-files:
+- README.md
+
+dependencies:
+  - base >= 4.7 && < 5
+  - classy-prelude
+  - persistent
+  - persistent-template
+
+library:
+  source-dirs:        src
+  default-extensions:
+    - NoImplicitPrelude
+    - OverloadedStrings
```

Comment:
`NoImplicitPrelude` is useful when you want to use an alternate Prelude. Yesod uses the `classy-prelude` by default, which is really nice.

Reply:
OverloadedStrings allow a string literal to be a `Text` or `ByteString` instead of just a `String`, which is really nice.